go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cockroachdb/examples-orms
 go 1.13
 
 require (
-	github.com/cockroachdb/cockroach-go/v2 v2.0.6
+	github.com/cockroachdb/cockroach-go/v2 v2.0.7-0.20200903114400-4bf206d91533
```

Comment:
Why can't this use a normal tag?

Reply:
It does now. When I opened the PR, the PR in cockroach-go hadn't merged yet.